.ci/gitlab/test.yml

```diff
@@ -71,7 +71,7 @@ cosim:unittests:
 prelude:unittests:
   extends: .test-common
   script:
-    - cabal v2-run clash-prelude:unittests
```

Comment:
In this line, you add an option terminator (`--`), but not in the others. Shouldn't they all need it? Could you please make it consistent regardless?

Reply:
They should be added to the others. In fact, it causes CI failures now.